lib/build.rb

```diff
@@ -66,6 +66,13 @@ def query_branches
       end
       $logger.debug("Querying potential build: #{b.name}")
       branch_details = github_query(@client) { @client.branch(@repository, b.name) }
+      skip_message_present = false
+      begin
+        skip_message_present = branch_details.commit.message['[decent_ci_skip]']
+      rescue
+        # Ignored
```

Comment:
If the commit for some reason doesn't have a commit message or something else goes awry just ignore it and leave the skip flag as false.